.werft/jobs/build/deploy-to-preview-environment.ts

```diff
@@ -99,18 +99,18 @@ export async function deployToPreviewEnvironment(werft: Werft, jobConfig: JobCon
         withVM,
     };
 
-    exec(`kubectl --namespace keys get secret host-key -o yaml > /workspace/host-key.yaml`)
+    exec(`kubectl --kubeconfig ${CORE_DEV_KUBECONFIG_PATH} --namespace keys get secret host-key -o yaml > /workspace/host-key.yaml`)
```

Comment:
To be on the 100% safe side it would be nicest to quote the path, so it is `--kubeconfig "${CORE_DEV_KUBECONFIG_PATH}"`. This goes for all of the places where we are using a path in an `exec` call